baked_in.go

```diff
@@ -213,7 +215,8 @@ func isOneOf(fl FieldLevel) bool {
 		panic(fmt.Sprintf("Bad field type %T", field.Interface()))
 	}
 	for i := 0; i < len(vals); i++ {
-		if vals[i] == v {
+		val := strings.Replace(vals[i], "'", "", -1)
```

Comment:
instead of separating this logic from the `parseOneOfParam2` function is it possible to do the replace before adding to the `oneofValsCache` cache, then the output of the function is constent and all other validations can benefit from the same changes 😄  eg. `required_with*`

Reply:
I'll give this a shot 👍